classroom_class.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :lable
+
```

Comment:
- [ ] Good job adding the `attr_accessor` method @zbahati, but alongside that, we need to have `attr_reader` for the `students` attribute. Please consider adding this one.